app/settings/globals.py

```diff
@@ -2,19 +2,40 @@
 from typing import Optional
 
 from starlette.config import Config
-from starlette.datastructures import Secret
+from starlette.datastructures import Secret, CommaSeparatedStrings
 
 from ..models.pydantic.database import DatabaseURL
 
-p: Path = Path(__file__).parents[2] / '.env'
+p: Path = Path(__file__).parents[2] / ".env"
 config: Config = Config(p if p.exists() else None)
 
-DATABASE: str = config('DATABASE', cast=str)
-DB_USER: Optional[str] = config('DB_USER', cast=str, default=None)
-DB_PASSWORD: Optional[Secret] = config('DB_PASSWORD', cast=Secret, default=None)
-DB_HOST: str = config('DB_HOST', cast=str, default='localhost')
-DB_PORT: int = config('DB_PORT', cast=int, default=5432)
-DATABASE_CONFIG: DatabaseURL = DatabaseURL(drivername="asyncpg", username=DB_USER, password=DB_PASSWORD, host=DB_HOST,
-                                           port=DB_PORT, database=DATABASE)
-ALEMBIC_CONFIG: DatabaseURL = DatabaseURL(drivername="postgresql+psycopg2", username=DB_USER, password=DB_PASSWORD,
-                                          host=DB_HOST, port=DB_PORT, database=DATABASE)
+DATABASE: str = config("DATABASE", cast=str)
```

Comment:
You'd do much better to use `BaseSettings` overriding all this for tests will be difficult.

Reply:
Re: architecture - I'm using Starlette's config, for separation of code, as detailed here: https://www.starlette.io/config/

For environ overrides: https://www.starlette.io/config/#reading-or-modifying-the-environment